uml/e66519fc27225a6f5fab4d4af6c6d64e.uml

```diff
@@ -0,0 +1,25 @@
+@startuml
```

Comment:
should this file be part of this blog post?

Reply:
nope, I'll remove it :smile: